x/ibc/testing/coordinator.go

```diff
@@ -50,84 +53,210 @@ func (coord *Coordinator) IncrementTime() {
 // CommitBlock commits a block on the provided indexes and then increments the global time.
 //
 // CONTRACT: the passed in list of indexes must not contain duplicates
-func (coord *Coordinator) CommitBlock(chains ...uint64) {
-	for _, index := range chains {
-		chain := coord.Chains[index]
+func (coord *Coordinator) CommitBlock(chains ...string) {
+	for _, chainID := range chains {
+		chain := coord.Chains[chainID]
```

Comment:
check for error, applies for all the `coord.Chains[chainID]`

```suggestion
		chain, ok := coord.Chains[chainID]
		require.True(t, ok, fmt.Sprintf("%s chain doesn't exist", chainID))
```

Reply:
you can wrap it on a function too 👍